app/lib/actions/katello/actions_jail.rb

```diff
@@ -0,0 +1,33 @@
+module Actions
+  module Katello
+    module ActionsJail
+      def content_view_id
+        input['content_view']['id']
+      end
+
+      def content_view_name
+        input['content_view']['name']
+      end
+
+      def content_view_label
+        input['content_view']['label']
+      end
+
+      def organization_id
+        input['organization']['id']
+      end
+
+      def organization_name
+        input['organization']['name']
+      end
+
+      def organization_label
+        input['organization']['label']
+      end
+      class Jail < ::Actions::ObservableAction::Jail
+        allow :content_view_id, :content_view_name, :content_view_label,
+              :organization_id, :organization_name, :organization_label
+      end
+    end
+  end
+end
```

Comment:
I am wondering how a concern will work in this case, class Jail is defined via the concern and then redefined later. I guess I need to test it.

By the way is the reason you inherit `::Actions::ObservableAction::Jail` instead of `Safemode::Jail` to also have `task` allowed?

Reply:
Well, I guess for actions we want have `task` object available to get some info. The first action that has that is `host_run_job_succeeded`, so I thought it might be useful for all the actions, thus I inherit `::Actions::ObservableAction::Jail`.

Also, I've updated the way the `Jail` gets possibility to be re-opened after the module is included.